lib/ui/view/workspace/workspace_view/workspace_viewmodel.dart

```diff
@@ -15,10 +17,23 @@ class WorkspaceViewModel extends BaseViewModel {
   final connectivityService = locator<ConnectivityService>();
   final storageService = locator<SharedPreferenceLocalStorage>();
   final api = WorkSpaceApi();
+  late BuildContext context;
```

Comment:
Viewmodel should never know anything about context or import to `material` or `Cupertino` , the snackbar can be gotten from stacked_services, register the service then make use of it in the viewmodel.

Reply:
Fixed, Custom Snackbar has been swapped for Snackbar Service by stacked